app/javascript/dashboard/routes/dashboard/settings/integrations/useHook.js

```diff
@@ -0,0 +1,26 @@
+import { computed, ref } from 'vue';
+import { useStore } from 'dashboard/composables/store';
+
+export const useHook = (integrationObj, integrationId) => {
```

Comment:
Let's also rename this to `useIntegrationHook`, `useHook` can be a bit confusion